src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,198 @@ description: Notifications are messages that communicate information to the user
 
 <Row>
 <Column colLg={8}>
 
-![High vs. low contrast](images/notification-usage-4.png)
+![Example of inline notification placement](images/notification-placement-inline.png)
+
+<Caption>
+  Example of an inline notification supporting an error message on a form
+</Caption>
 
 </Column>
 </Row>
 
-## Placement
+### Behavior
 
-#### Toast notifications
+#### Dismissal
 
-Toast notifications slide in and out a page from the top-right corner. Actionable notifications do not appear on mobile screen widths.
+Inline notifications do not dismiss automatically. They persist on the page until the user dismisses them or takes action that resolves the notification.
+
+A small "x" in the top right corner is used to dismiss inline notifications. Including the close button is optional and should not be included if it is critical for a user to read or interact with the notification.
+
+## Toast notifications
+
+### Formatting
 
 <Row>
 <Column colLg={8}>
 
-![Toast notification example](images/notification-usage-2.png)
+![Example of toast notification with action button and time stamp](images/notification-toast-action.png)
 
 </Column>
 </Row>
 
-#### Inline notifications
+#### Action
+
+Toast notifications can include a link at the end of their body content. This link should be short and navigate users to a page or modal where they can take action to address the notification or find further information. Because toast notifications automatically dismiss, there should be alternative routes to navigate to the link destination.
+
+If you include links within the notification body that redirect the user to steps, be sure keyboard and screenreader users have enough time to access the notification or can easily find the notification after it disappears.
+
+#### Time stamp
+
+Toast notifications include a time stamp at the bottom the container. The time stamp shows the time the notification was sent. Using time stamps is optional but toasts should be consistent across the product so either all toasts should include time stamps or none of them should. The time stamp is optional and can be removed if a third line of content is needed.
+
+#### Sizing
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+Toast notifications use a fixed width and their height depends on the length of the notification message. As noted in the content guidelines, limit toast notifications to two lines of text.
+
+#### Placement
+
+Toast notifications slide in and out from the top right of the screen. They align to the edge of the last column and can stack with `$spacing-03` in-between. New toast notifications should appear at the top of the list, with older notifications being pushed down until they are dismissed.
```

Comment:
What happens if the height of the stacked notifications is greater than the browser height?

Notifications are typically absolutely positioned, as a result, how would one implement this guidance:

> They align to the edge of the last column

Reply:
Clarification on @joshblack's last point; "absolute positioning" means that an element doesn't reference other elements, so the "align to the edge of the last column" is difficult/impossible to apply from a code perspective.

Do we need to either address the design guidance, or address the development implementation?

Reply:
This is a good point that we don't currently have a solution for. This will need to be opened as a issue to figure out. I also think if they're disappearing after x seconds the chances of there being a long stack of them seems very unlikely.

Reply:
@joshblack do you have a suggestion for how to provide the positioning guidance? I was trying to address this issue https://github.com/carbon-design-system/carbon-website/issues/76

Reply:
@jillianhowarth it seems like we could re-word it to say that the notifications are aligned vertically and use the spacer in between and not mention the edge alignment? Just a thought 👀